src/org/omegat/util/Platform.java

```diff
@@ -87,6 +92,8 @@ public enum OsType {
         isWindows = osName.startsWith("windows");
         isMacOS = osName.startsWith("mac");
         isLinux = osName.startsWith("linux");
+        isBSD = osName.startsWith("bsd");
+        isUnix = isLinux || isBSD;
```

Comment:
`isUnixLike` is better abstraction. see https://github.com/search?q=isUnixLike&type=code

```suggestion
        isUnixLike = isLinux || isBSD;
```

Reply:
should I `git push --force` the fix, or simply commit another one with the fix?

Reply:
It is ok to simply commit another one. I will squash and merge when all is ok.